go/test/endtoend/vtorc/api/api_test.go

```diff
@@ -91,6 +91,22 @@ func TestAPIEndpoints(t *testing.T) {
 		return response != "null"
 	})
 
+	t.Run("Dynamic Configuration", func(t *testing.T) {
+		// Get configuration and verify the output.
+		status, resp, err := utils.MakeAPICall(t, vtorc, "/api/config")
+		require.NoError(t, err)
+		assert.Equal(t, 200, status)
+		assert.Contains(t, resp, `"snapshottopologyinterval": 0`)
```

Comment:
Isn't the response JSON? I'm wondering why the case of the config key is not being preserved. I'd expect it to be `SnapshotTopologyInterval`

Reply:
Yeah, but for some reason getting all the settings from the config, it returns everything in small case. Must be some internal implementation sideeffect of viper.